<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyKeyDialog</class>
 <widget class="QDialog" name="MyKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>270</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 114, 127);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_window">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>@</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>[</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_28">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>abc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
background-color: rgb(139, 139, 139);</string>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
